### ISSUE TITLE
Extend the password forgot token lifetime

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#9986]
From https://github.com/mozilla/fxa/blob/94ffcb4b3d8ccb6f853ca61716999834129ceda2/packages/fxa-auth-server/config/index.ts#L588 our forgot token expires in 1hr. While I suspect that this is enough time for most users there will probably be some that will click an expired link.

Since the `passCode` is pretty large and a user can only attempt to use it 3 times, we might be able to safely extend this to 6-12hrs.

┆Issue is synchronized with this [Jiraserver Task](https://mozilla-hub.atlassian.net/browse/FXA-3812)

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Stéphanie Ouillon do you have an opinion here?  Do you want to set up a meeting for more information?

**Comment by data-sync-user** [comment]
➤ Stéphanie Ouillon commented:

Is there a particular reason to extend the current time? If a user is trying to connect and would like to reset their password,  why they would wait for 6 hours before clicking on the link?

Currently, what happens for people who do take such time and gets an expired link, can they ask for a new link?

**Comment by data-sync-user** [comment]
➤ Vijay Budhram commented:

This is actually to support a new feature that allows users to set a password on their account after the fact. Ref [FXA-3486] 

To reduce complexity of feature, we planned on reusing our password reset tokens. We are not sure how long it will take these users to finish the account setup though. The 6hr-12hrs estimate was based on how long we allow new accounts to skip signin confirmation. 

Currently for reset password, if the link is expired, they can request a new one. Unforunately, this UX is specific to password reset.

**Comment by data-sync-user** [comment]
➤ Stéphanie Ouillon commented:

Vijay Budhram  Got it thanks, I remember we talked about it in a RRA: https://docs.google.com/document/d/1_isJF7p03PTYjsb6lsWEreFnSZjwXOaWfj8gutUq1UM/edit# ( https://docs.google.com/document/d/1_isJF7p03PTYjsb6lsWEreFnSZjwXOaWfj8gutUq1UM/edit#|smart-link )

Extending the lifetime of the forgot token seems reasonable in that case.

I’d like to double check something here: 

Back then, I wrote down in the notes of the RRA:

  * The verification link is sent by email for the user to complete their setup. This is actually using the same code as to ‘reset’ a password.
  * As for other verification links, the codes never expire. If the user loses them, they can reset the account.

The codes of the verification links are separate from the forget token, right?

What would happen if the user missed the 6hr-12hrs window?
